packages/go_router/example/lib/auth_bloc/bloc/authentication_event.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of 'authentication_bloc.dart';
```

Comment:
nit: our style guide discourages the use of `part of`.

Reply:
(https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-library-and-part-of)

Reply:
solved. please check again.